13-microbiome-exploration.Rmd

```diff
@@ -31,71 +31,133 @@ Investigating the prevalence of microbes allows you either to focus on changes,
 which pertain to most of the samples, or to focus on less often found microbes,
 which are nonetheless abundantly found in a small number of samples.
 
-Population prevalence (frequency) at a 1% relative abundance threshold:
+On the raw data, the population prevalence (frequency) at a 1% relative
+abundance threshold (`detection = 1/100` and `as_relative = TRUE`), can look
+like this. The low prevalence in this examples can be explained by rather
```

Comment:
I am not sure what "low prevalence" refers to in here since 1% relative abundance threshold is in fact rather strict for most taxa. I would remove the comment low prevalence, or add some explanation in what sense the prevalence is small. Or we can change the threshold to a less conservative 0.1%

Reply:
the low prevalence of the samples aka. the result of `getPrevalence`. 0.2 and 0.3 is rather low, wouldn't you agree?

Reply:
"Low" would indicate that the prevalences are lower than we would expect based on a given detection threshold. But in my experience the values are here not specifically low but instead expected when using a conservative detection threshold such as 1% relative abundance.

Reply:
I  think in this instance, the situation is different. `GlobalPatterns` contains many different sample types, so that most prevalent ASV has prevalance around `0.3`. I regard this as low, since most of the time if analyzing closely related samples at least a few ASV have a prevalence close to `1.0`. Is my reasoning sound? How would you phrase it differntly so make this point clear?

Reply:
Indeed, heterogeneity in samples makes a difference. Hmm.. but even in that case, isn't this level of prevalence then "as expected" rather than "low"..  It makes more sense now anyway. Low is partially subjective evaluation in this case, one option is to remove but for now, I do not have strong preferences.

Reply:
So we leave for now as it is and I merge it?

Reply:
Fine. I may suggest modification at some later revision stage if it still confuses me at that point.